File: app/src/main/java/me/manabu/utils/DpUtils.java
package me.manabu.utils;

import android.content.Context;
import android.util.TypedValue;

public class DpUtils {
    public static float fromDpToPixels(Context context, float dp){
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.getResources().getDisplayMetrics());
    }
}

File: app/src/main/java/me/manabu/utils/DpUtils.kt
package me.manabu.utils

import android.content.Context
import android.util.TypedValue

object DpUtils {
    fun fromDpToPixels(context: Context, dp: Float): Float {
        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.resources.displayMetrics)
    }
}
